metatron-app/metatron-discovery
# #2342 fix kpi chart value and spelling mistake

### Description
1. Fix the incorrect display of KPI chart values if user-defined columns exist (#2342)
2. KPI chart spelling mistake (#2334)

**Related Issue** :  (#2342, #2334)



### How Has This Been Tested?
1. Create a KPI Chart with user-defined columns.
2. Check 'Previous' spelling
![image](https://user-images.githubusercontent.com/45194478/61025959-1cf67000-a3ed-11e9-8d89-27e296eda01a.png)



#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
